src/pipx/util.py

```diff
@@ -75,15 +75,11 @@ def get_venv_paths(root: Path) -> Tuple[Path, Path]:
         return bin_path, python_path
 
 
-def get_script_output(interpreter: Path, script: str, *args: Union[str, Path]) -> str:
-    proc = run_subprocess([interpreter, "-c", script, *args], capture_stderr=False)
-    return proc.stdout
-
-
 def get_site_packages(python: Path) -> Path:
-    output = get_script_output(
-        python, "import sysconfig; print(sysconfig.get_path('purelib'))"
-    )
+    output = run_subprocess(
+        [python, "-c", "import sysconfig; print(sysconfig.get_path('purelib'))"],
+        capture_stderr=False,
+    ).stdout
```

Comment:
Is there a behaviour change in this part? I assumed there is because there’s significant code diff, but couldn’t figure out what changed reading it multiple times.

Reply:
In the original, we submitted the entire contents of `venv_metadata_inspector.py` as one giant string as the argument after `python -c`.

The version I changed it to uses the full-path to `venv_metadata_inspector.py` file to run it like a normal script, e.g. `python venv_metadata_inspector.py`.

Reply:
Sorry, I made a previous comment referring to the wrong code.

No, this particular code should be exactly the same.  I just took the code that used to be wrapped inside of get_script_output.  It should be functionally the same, just not wrapped in another function.

Reply:
This was the only remaining place using `get_script_output` so I thought it might be simpler just to show the code instead of using one more level of function.

Reply:
Cool, I just wanted to be sure I don’t miss any changes :+1: